package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/main.js",
   "scripts": {
     "build": "rollup --config",
-    "start": "webpack-dev-server --mode development",
+    "start": "webpack serve --mode development",
```

Comment:
after a little research a found this [solution](https://stackoverflow.com/questions/40379139/cannot-find-module-webpack-bin-config-yargs)
![image](https://user-images.githubusercontent.com/29656850/115025349-a034d780-9e97-11eb-9807-db371cd9b1f6.png)

Reply:
Please, move this change to another PR, because it is not related to the feature proposed here